packages/console/src/pages/Users/index.tsx

```diff
@@ -114,12 +122,14 @@ const Users = () => {
           ))}
         </tbody>
       </table>
-      {totalCount && (
-        <Pagination
-          pageCount={Math.ceil(totalCount / pageSize)}
-          pageIndex={pageIndex}
-          onChange={setPageIndex}
-        />
+      {conditional(
```

Comment:
`conditional` is unnecessary here

Reply:
`totalCount` can be zero, then will render a `0`. And it also can be "undefined", so `totalCount > 0` is not fit too. Then comes to `conditional`.

Reply:
`!!totalCount &&` or `Boolean(totalCount) &&` ?

Reply:
`conditional` would be more clear? Not sure if is a good practice to use `conditional` in JSX though.

Reply:
removed.